browser-tests/functional-tests/functional.spec.js

```diff
@@ -1,5 +1,9 @@
 const { test, expect } = require("@playwright/test");
-const TEST_CONSTANTS = require("../../test/constants");
+
+const HTTP_HEADER_USER_AGENT_ANDROID =
+  "Mozilla/5.0 (Linux; Android 8.0.0; Nexus 5X Build/OPR6.170623.013) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.98 Mobile Safari/537.36";
+const HTTP_HEADER_USER_AGENT_IPHONE =
+  "Mozilla/5.0 (iPhone; CPU iPhone OS 10_3 like Mac OS X) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.0 Mobile/14E277 Safari/602.1";
```

Comment:
These are duplicated from the unit test constants, for the sake of setting up TS cross-compatibility, it feels better to instead break the dependency on `src`